apps/vast/pyvast_threatbus/pyvast_threatbus.py

```diff
@@ -385,15 +386,15 @@ async def ingest_vast_ioc(vast_binary: str, vast_endpoint: str, indicator: Indic
     @param vast_endpoint The endpoint of a running vast node ('host:port')
     @param indicator The STIX-2 Indicator to query VAST for
     """
-    global logger
+    global logger, matcher_name
     vast_ioc = indicator_to_vast_matcher_ioc(indicator)
     if not vast_ioc:
         logger.error(
             f"Unable to convert STIX-2 Indicator to VAST compatible IoC. Is it a point IoC? {indicator}"
         )
         return
     vast = VAST(binary=vast_binary, endpoint=vast_endpoint, logger=logger)
-    proc = await vast.import_(type="intel.indicator").json().exec(stdin=vast_ioc)
+    proc = await vast.matcher().add(matcher_name, indicator.value, indicator.reference).exec()
```

Comment:
Should this instead open a `matcher import` command once and then just feed it data via stdin here?

Reply:
I think I'd prefer to go with the simplest solution for now until we hit performance issues; using stdin I'm not sure if we would need additional error handling/retrying for the case that VAST restarts.